app/scripts/migrations/099.ts

```diff
@@ -0,0 +1,56 @@
+import { cloneDeep, isEmpty } from 'lodash';
+import { v4 as uuid } from 'uuid';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 99;
```

Comment:
Is it worth delaying this PR so we can nab the 100th migration? 😄

Reply:
I've started at 97 and after couple rebases 99. If we hold it a bit more we gonna snipe it :D